lab-kaylyn/app/view/home/_home.scss

```diff
@@ -0,0 +1,32 @@
+@import "custom-bootstrap";
+@import "theme";
+
+html, body {
+  width: 100%;
+  height: 100%;
+}
+
+.home {
+
+  .gallery-list-container {
+    background-color: $app-white;
+    font-size: 1.5em;
+    text-align: center;
+
+    ul {
+      padding: 0;
+    }
+
+    li {
+      list-style: none;
+    }
+
+    h2 {
+      font-size: 1em;
+    }
+
+    p {
+      font-size: .75em;
+    }
```

Comment:
@kaylynyuh may be best to separate these styles into a single `base` partial - these are almost single element resets